test/integration_test/test-deploy.sh

```diff
@@ -105,9 +105,11 @@ KUBEVERSION=$(kubectl version -o json | jq ".serverVersion.gitVersion" -r)
 sed -i 's/<kube_version>/'"$KUBEVERSION"'/g' /specs/stork-scheduler.yaml
 sed -i 's|'openstorage/stork:.*'|'"$image_name"'|g'  /specs/stork-deployment.yaml
 
-if [ "$volume_driver" != "pxd" ] ; then
-	sed -i 's/- --driver=pxd//g' /testspecs/stork-test-pod.yaml
+# Replace volume driver in stork
+if [ "$volume_driver" != "" ] ; then
+	sed -i 's/- --driver=pxd/- --driver='"$volume_driver"'/g' /specs/stork-deployment.yaml
```

Comment:
```suggestion
	sed -i 's/- --driver=pxd//g' /specs/stork-deployment.yaml
```
Need to remove the driver name actually